packages/nimble-components/src/wafer-map/index.ts

```diff
@@ -0,0 +1,67 @@
+import { attr, observable } from '@microsoft/fast-element';
+import { DesignSystem, FoundationElement } from '@microsoft/fast-foundation';
+import { template } from '../theme-provider/template';
+import { styles } from './styles';
+import {
+    WaferMapColorsScale,
+    WaferMapDie,
+    WaferMapOrientation,
+    WaferMapQuadrant
+} from './types';
+
+declare global {
+    interface HTMLElementTagNameMap {
+        'nimble-wafer-map': WaferMap;
+    }
+}
+
+/**
+ * A nimble-styled WaferMap
+ */
+export class WaferMap extends FoundationElement {
+    @attr({
+        attribute: 'quadrant'
+    })
+    public quadrant: WaferMapQuadrant = WaferMapQuadrant.topLeft;
+
+    @attr({
+        attribute: 'orientation'
+    })
+    public orientation: WaferMapOrientation = WaferMapOrientation.top;
+
+    @attr({
+        attribute: 'max-characters'
+    })
+    public maxCharacters = 4;
+
+    @attr({
+        attribute: 'show-die-labels'
+    })
+    public showDieLabels = true;
+
+    @attr({
+        attribute: 'die-labels-suffix'
+    })
+    public dieLabelsSuffix = '';
+
+    @attr({
+        attribute: 'is-continuous'
+    })
+    public isContinuous = true;
+
+    @attr({
+        attribute: 'highlighted-values'
+    })
+    public highlightedValues: number[] = [];
```

Comment:
A complex datatype like an array should not have an html attribute unless we choose a well-defined string representation like JSON, etc. For now we can just make it an observable property and add an attribute representation if it makes sense in the future:

```suggestion
    @observable public highlightedValues: number[] = [];
```